src/App.tsx

```diff
@@ -1,5 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 export default function App() {
-  return <p>Webpack + TypeScript + React = ❤️</p>;
+  const [val, setVal] = useState(0);
+  const add = () => {
+    setVal(val + 1);
+  };
+  return <div>
+    <p>Webpack + TypeScript + React = ❤️</p>
+
+    {'1. click add; 2. Change this text to see hmr: testText'}
```

Comment:
I think HMR is already enabled by webpack-dev-server. What is the profit of `@pmmmwh/react-refresh-webpack-plugin`?